index.coffee

```diff
@@ -1,7 +1,7 @@
 MealzClient = require "./mealz_client"
 HttpClient = require 'scoped-http-client'
 
-client = new MealzClient(process.env.HUBOT_MEALZ_URL, HttpClient)
+client = new MealzClient("https://localhost:3000/api/", HttpClient)
```

Comment:
Wups! :D I'll fix it in a minute :)

Groetjes,
Lewis




On Thu, May 26, 2016 at 12:39 AM -0700, "Bob Van Landuyt" <notifications@github.com> wrote:












In index.coffee:
> @@ -1,7 +1,7 @@
>  MealzClient = require "./mealz_client"
>  HttpClient = require 'scoped-http-client'
>  
> -client = new MealzClient(process.env.HUBOT_MEALZ_URL, HttpClient)
> +client = new MealzClient("https://localhost:3000/api/", HttpClient)



Hmmm, I think you forgot something here :-)



—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub